gluster/block/README.md

```diff
@@ -1,31 +1,37 @@
 # glusterblock Volume Provisioner for Kubernetes 1.5+
 
-
 [![Docker Repository on Quay](https://quay.io/repository/external_storage/glusterblock-provisioner/status "Docker Repository on Quay")](https://quay.io/repository/external_storage/glusterblock-provisioner)
+
+Latest stable container image can be pulled from:
+
 ```
 quay.io/external_storage/glusterblock-provisioner:latest
 ```
 
-# Test instruction
+# Build Gluster Block Provisioner
+
+If you have `go` envrionment configured, you can build your own binary or container image as shown below:
 
-* Build glusterblock-provisioner and container image
+#### Clone and build binary and docker image
 
 ```bash
-[root@localhost]#go build glusterblock-provisioner.go
-[root@localhost]#docker build -t glusterblock-provisioner .
+[root@localhost]# git clone https://github.com/kubernetes-incubator/external-storage
+[root@localhost]# cd external-storage/gluster/block/cmd/glusterblock-provisioner
+[root@localhost]# go build glusterblock-provisioner.go
+[root@localhost]# docker build -t glusterblock-provisioner .
 ```
 
-* Start Kubernetes local cluster
+#### Start Kubernetes local cluster
```

Comment:
level 4 ?
also, why is this sorted in under level-1 headline "Build Gluster Block Provisioner"? This does not seem logical.